include/abb_librws/rws_interface.h

```diff
@@ -49,6 +49,113 @@ namespace rws
 class RWSInterface
 {
 public:
+  /**
+   * \brief Type of a mechanical unit.
+   */
+  enum MechanicalUnitType
+  {
+    NONE,      ///< The unit has no type.
+    TCP_ROBOT, ///< The unit is a TCP robot.
+    ROBOT,     ///< The unit is a robot.
+    SINGLE,    ///< The unit is a single.
```

Comment:
A single what?

Reply:
Clarified in 44e45c4

A `SINGLE` is defined as a mechanical unit with only one joint.